{{ cookiecutter.package_name }}/docs/documentation/installation.rst

```diff
@@ -0,0 +1,54 @@
+.. _{{ cookiecutter.module_name }}-installation:
+
+============
+Installation
+============
+
+|Code Size|
+
+**********
+With `pip`
+**********
+
+.. container::
+
+	|PyPI| |PyPI Format|
+
+The easiest way to get *{{ cookiecutter.module_name }}* is to install with `pip <https://pypi.org/project/{{ cookiecutter.module_name }}/>`_. To install with pip
+::
+
+    pip install {{ cookiecutter.module_name }}
+
+
+***********
+From source
+***********
+
+To install the latest version, or from a branch, *{{ cookiecutter.package_name }}* can be installed from `source <https://github.com/{{ cookiecutter.github_project }}.git>`_.
+
+From the command line.
+
+.. code-block:: bash
+   :caption: setup.py
+   :name: setup-py
```

Comment:
```suggestion
```

Reply:
Why delete these, doesn't it enable RST linking?